protostar.toml

```diff
@@ -1,5 +1,5 @@
 ["protostar.config"]
-protostar_version = "0.3.1"
+protostar_version = "0.3.2"
```

Comment:
careful, it also in the other PR, you will have conflict during the merge. 
I will validate this PR (wating to answer to other), so I recommand to merge with main repo before PR in other PR.